### DISCUSSION TITLE
Signal data file of 40 MHz BW and 15KHz SCS  of no noise is giving SSS constellation as if noise is present

### DISCUSSION THREAD

**Discussion by amulamoottil** [catkira/py3gpp#35] [Q&A]
Signal data file of 40 MHz BW and 15KHz SCS  of no noise is giving SSS constellation as if noise is present. I have updated that nrB = 216 instead of 240 for this case and fft 4096 instead of 2048 for fft. I am getting the correct cell ID but still noise is being present in the values. Can you tell me why?

**Comment by amulamoottil** [comment]
@catkira Please note that the above signal is generated using matlab

**Reply by catkira** [reply]
can u provide the file?

**Reply by amulamoottil** [reply]
[test_signal_40MHz_15KHz_cellid4.zip](https://github.com/catkira/py3gpp/files/11790319/test_signal_40MHz_15KHz_cellid4.zip)
Here you go this is the matlab generated signal

**Reply by amulamoottil** [reply]
@catkira  Did you get any findings as I tried with a 20Mhz and 15scs signal generated from Matlab and same issue was difference was seen

**Reply by amulamoottil** [reply]
Hi @catkira , Can you let me know?

**Reply by catkira** [reply]
I recommend that You compare the demodulation with Matlab and find where differences appear.
